providers/os/connection/snapshot/volumemounter.go

```diff
@@ -26,12 +26,12 @@ type VolumeMounter struct {
 	// where we tell AWS to attach the volume; it doesn't necessarily get attached there, but we have to reference this same location when detaching
 	VolumeAttachmentLoc string
 	opts                map[string]string
-	cmdRunner           *LocalCommandRunner
+	CmdRunner           *LocalCommandRunner
 }
 
 func NewVolumeMounter(shell []string) *VolumeMounter {
```

Comment:
we can move this entirely to `device/linux` once we migrate the snapshot connections (AWS, Azure, GCP)